jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -310,8 +313,8 @@ def vue_do_aper_phot(self, *args, **kwargs):
                 sum_ct_err = None
 
             if include_flux_scale:
-                flux_scale = flux_scale * rawsum.unit
-                sum_mag = -2.5 * np.log10(rawsum / flux_scale) * u.mag
+                flux_scale = flux_scale * phot_table['sum'][0].unit

```

Comment:
`phot_table` entry had the per-steradian taken out, but not `rawsum` because previously we assume whatever scaling provided would be in the *data unit*, which is no longer true now for this JWST use case. My commit should have fixed Ori's problem.